src/actions.ts

```diff
@@ -259,9 +272,8 @@ export const go = <Cell: Types.CellBase>(
   };
 };
 
-export const modifyEdge = (field: $Keys<Types.Point>, delta: number) => (
-  state: Types.StoreState<*>,
-  event: *
+export const modifyEdge = (field: keyof Types.IPoint, delta: number) => (
+  state: Types.IStoreState<any>
```

Comment:
The removal of the `event` param may break usages. Likely should verify that nothing could pass the event through here that would have argument issues.